temel_bilgiler/kurulumlar/web_nasıl_çalışır.md

```diff
@@ -0,0 +1,57 @@
+### Giriş
+
+Web'i nasıl programlayacağınızı anlamadan önce, web'in kendisini muhtemelen şimdi sahip olduğunuzdan daha titiz bir şekilde anlamanız gerekir. Bu kavramlar, içinde çalışacağınız ekosistem hakkında daha bütünsel bir anlayış sağlar ve diğer geliştiricilerle işiniz hakkında akıllıca konuşmanıza olanak tanır.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konuların genel bir özetini içerir.
+
+* İnternetin ne olduğunu açıklayınız.
+* Paketlerin ne olduğunu ve veri aktarımı için nasıl kullanıldığını açıklayınız.
+* Bir web sayfası, web sunucusu, web tarayıcısı ve arama motoru arasındaki farkları anlayınız.
+* İstemcinin ne olduğunu kısaca açıklayınız.
+* Sunucu nedir kısaca açıklayınız.
+* IP adreslerinin ne olduğunu açıklayınız.
+* DNS sunucularının ne olduğunu açıklayınız.
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+  1. İnternetin nasıl çalıştığına dair genel bir bakış için bu [BBC short](https://www.youtube.com/watch?v=eHp1l73ztB8) 'u izleyin.  
+  2. Mozilla'nın "İnternet nasıl çalışır?" başlıklı bu [makalesini](https://developer.mozilla.org/en-US/Learn/Common_questions/How_does_the_Internet_work) okuyun.  
+  3. [How the Internet Works in 5 Minutes](https://youtu.be/7_LPdttKXPc?t=46s) izleyin.  
+  4. Web sayfası, web sunucusu ve arama motoru arasındaki [farkları](https://developer.mozilla.org/en-US/Learn/Common_questions/Pages_sites_servers_and_search_engines) okuyun.  
+  5. Web tarayıcısının ne olduğunu açıklayan bu [Google](https://youtu.be/BrXPcaRlBqo) kısa videosunu izleyin. Ardından, [şu anda](https://www.whatsmybrowser.org/) hangi web tarayıcısını kullandığınızı öğrenin.  
+  6. Web'in bir bölümünün nasıl [diğeriyle etkileşime girdiğini](https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/How_the_Web_works#Clients_and_servers) ve bir DNS isteğini nasıl [okuduğunu](https://developer.mozilla.org/en-US/Learn/Common_questions/What_is_a_domain_name#How_does_a_DNS_request_work) veya [izlediğini](https://www.youtube.com/watch?v=72snZctFFtA&feature=youtu.be&t=45s) okuyun.  
```

Comment:
fix: 6. Web'in bir bölümünün [diğeriyle nasıl etkileşime girdiğini bu İngilizce yazıda okuyun](https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/How_the_Web_works#Clients_and_servers) ve bir DNS isteğinin nasıl çalıştığını bu İngilizce kaynaklardan ister [okuyun](https://developer.mozilla.org/en-US/Learn/Common_questions/What_is_a_domain_name#How_does_a_DNS_request_work), ister [izleyin](https://www.youtube.com/watch?v=72snZctFFtA&feature=youtu.be&t=45s).